ApollosProject/apollos-apps
# fix: Daily Scripture Navigation

## ✏️ Description
In a previous PR, we introduced a custom Tab Type for Daily Habits. This PR expands on that to resolve an issue where the Scripture Experience was not navigating to the correct component.

## 🔬 To Test
1. Run a local instance of the Apollos Cluster using [this PR](https://github.com/ApollosProject/apollos-cluster/pull/371#pullrequestreview-1181841396)
2. Run the Apollos app in a sim (`yarn && yarn start` then `yarn ios`)
3. Navigate to the "Daily" tab and scroll down until you see the "Scripture" card
4. Tap on it and you should see an actual Content Item pop up this time

## 📸 Screenshots
### Before the Fix
https://user-images.githubusercontent.com/45076058/202079450-0a62933a-edf4-4f2f-8824-fbe798e4c131.mp4

### After the Fix
https://user-images.githubusercontent.com/45076058/202079239-9355c9a0-90f9-4e94-a4c9-d4071295e36f.mp4


## ⚠️ To-do Before Merge

- [x] Ensure the [Cluster PR](https://github.com/ApollosProject/apollos-cluster/pull/371#pullrequestreview-1181841396) is merged
- [x] Ensure that #2766  is merged

## 📝 Checklist
